web/src/clipboard_handler.ts

```diff
@@ -0,0 +1,35 @@
+import * as blueslip from "./blueslip"; 
+import {try_stream_topic_syntax_text} from "./copy_and_paste";
+
+export function clipboard_handler($link_element:JQuery, hide_popover:()=>void):void {
+
+    if(typeof $link_element.data("clipboard-text")==="string"){
+        // eslint-disable-next-line @typescript-eslint/consistent-type-assertions
```

Comment:
had to disable eslint on next line because next line returns type any by default it is built in Jquery

Reply:
You can instead use `assert`s here. Look at other areas in the codebase where it is used.
In fact, there is no need to pass the link element here if all we do is extract one string from its dataset. We can directly pass that string here.